analyzer/src/main/kotlin/managers/utils/NuGetDependency.kt

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright (C) 2022 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
```

Comment:
The extracted code to me seems a little bit like cyclic dependency. I believe it needs refactoring 
and doing so I guess it's easier if it's not separate before refactoring. 

I'm good with all other commits, would you be fine splitting this out so we can merge the rest and investigate this commit separatly?

Reply:
Ah sorry, then I overlooked that - thought it was relatively easy. Thanks anyway.